docs/source/getting-started/index.rst

```diff
@@ -0,0 +1,96 @@
+.. Copyright (c) 2020 The DAML Authors. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _new-quickstart:
+
+Full Stack DAML
+###############
+
+**Disclaimer:** This guide is being actively developed.
+Expect major changes to the tutorial text and minor changes to the template application.
+
+The goal of this tutorial is to get you up and running with full-stack, DAML-driven app development. We will provide you with the template for a miniature social networking app, get you writing your first end-to-end feature and finally deploy your new app to a persistent ledger.
```

Comment:
Please don't use "DAML-driven" anymore. Just use "DAML". It's so much more than a programming language. 😃